### ISSUE TITLE
check G1 documentation

### ISSUE THREAD

**Issue by LinxuanHu** [ubsuny/23-Homework4G4#72]
can you check our documentation

**Comment by WildJimmy** [comment]
Yep, I'll look it over and let you know if anything jumps out at me

**Comment by WildJimmy** [comment]
@LinxuanHu I just checked it over. I captizalized/rewrote some of your headers to make the formatting look a little nicer and included it as a pull request here (https://github.com/ubsuny/23-Homework4G1/pull/88)

**Comment by LinxuanHu** [comment]
> @LinxuanHu I just checked it over. I captizalized/rewrote some of your headers to make the formatting look a little nicer and included it as a pull request here ([ubsuny/23-Homework4G1#88](https://github.com/ubsuny/23-Homework4G1/pull/88))

thanks, but i am not the maintainer of my group
